src/batched/KokkosBatched_Vector.hpp

```diff
@@ -138,13 +138,13 @@ namespace Kokkos {
 	
 	
     template<typename T, int l>
-    class ArithTraits<Vector<SIMD<Kokkos::complex<T>>,l> > { 
+    class ArithTraits<KokkosBatched::Experimental::Vector<KokkosBatched::Experimental::SIMD<Kokkos::complex<T>>,l> > { 
```

Comment:
No love for my template template, alas...

Reply:
Uhm that didn't compile without the change ...

Reply:
when I looked at the old code RealPart and ImagPart, I saw they were only for SIMD. That's why I did not try @mhoemmen 's solution template template.
Now, I have just tried it and it works. Should I create a new PR for this?

Reply:
@vqd8a I don't mean to tell you what to do :-)  If it needs to work for AVX, etc. as well as SIMD, then my approach would give you a generic solution.

Reply:
@crtrott @kyungjoo-kim should we need this generic solution?

Reply:
(You don't _need_ it, but you do need ArithTraits, etc. to work for `AVX<T>` and any other `SimdImplementation<T>` you want to support, as well as `SIMD<T>`.  However that happens is up to you :-) .)